crates/driver/src/domain/eth/mod.rs

```diff
@@ -96,7 +96,7 @@ impl From<EffectiveGasPrice> for U256 {
 ///
 /// https://eips.ethereum.org/EIPS/eip-2930
 #[derive(Debug, Clone, Default)]
-pub struct AccessList(HashMap<Address, HashSet<StorageKey>>);
+pub struct AccessList(HashMap<ContractAddress, HashSet<StorageKey>>);
```

Comment:
This is incorrect. Access lists can also warm non-contract addresses (for example if you transfer ETH to an EOA, you can include its address (without storage slots) to warm up the address and pay less for the transfer).